security/docker/Dockerfile.flexvolumedriver

```diff
@@ -1,9 +1,9 @@
-FROM ubuntu:xenial
+FROM alpine:3.7
```

Comment:
why that change ?

Reply:
It's recommended in flexvolume.

Reply:
do they say why ? we build everything based on xenial so far

but actually why not make it
FROM scratch 

as if it works on alpine when built on xenial; it means it's statically linked and thus you need nothing (well you will need a statically linked bash I guess to run your script)

Reply:
That's right, i need the bash file to run the script

Reply:
but the script doesn't do anything that couldn't be done outside - does it ?